src/cascade/input_data/configuration/form.py

```diff
@@ -53,7 +53,17 @@ def _full_form_validation(self, root):  # noqa: C901 too complex
             if mean is None and np.isinf(lower) or np.isinf(upper):
                 mean = max(lower, 0)
             std = self.std
-            nu = self.nu
+
+            if self.nu is None:
```

Comment:
Are there any comments around these parts about what "nu" is?  or "why" nu exists?

If not, if there is a short way to add a little bg to this chunk, it could be nice, I think.

Reply:
I don't think this is the place for that kind of documentation. I'll make a note in the code where the parameters are actually defined that points to Brad's documentation of how they are used.